projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/datajobs/SwaggerConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.datajobs;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.servers.Server;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+/**
+ * This class exists so that when running quickstart vdk the swagger ui has the correct base url.
+ *
+ * <p>Without this in qucikstart vdk the swagger will think the url should be localhost:80/.
+ *
+ * <p>This is not actually correct! We are mapping port 80 from the cluster port 8092 on the local
+ * machine. There is no way for the service to know this mapping is happening so we need to provide
+ * it with the correct url.
+ */
+@Configuration
+public class SwaggerConfiguration {
+
+  private final OpenAPI openAPI;
+
+  @Value("${swagger-ui.hostname:}")
```

Comment:
Currently, we document all variables in application-prod.yaml 
It could be just 

```
# This is ... 
# ... 
# ...
swagger-ui.hostname=${SWAGGER_UI_HOSTNAME:""}


It has the following benefits 
-  For people not familiar with spring auto wiring from env variables it makes it clear what is the env variable name that is used.
- It acts as a single place to document all variables used by the Control service.


As part of the variable documentation, please explain when (in what deployment topology) a user should use it (set it). It is currently not clear to me.

Reply:
details added to the properties file. 
Let me know if you are happy with it.